tests/k8s_cmdline_tests.yaml

```diff
@@ -357,6 +357,24 @@
     --base_resource_list k8s_testcases/example_podlist/pods_list.json
   expected: 1
 
+- name: livesim_test_all_dot
+  args: >
+    --connectivity livesim_tests/
+    --ns_list livesim_tests/
+    --pod_list livesim_tests/
+    --output_format dot/
+    --expected_output expected_cmdline_output_files/livesim_test_all_dot.txt
+  expected: 0
+
+- name: livesim_test_all_txt
+  args: >
+    --connectivity livesim_tests/
+    --ns_list livesim_tests/
+    --pod_list livesim_tests/
+    --output_format txt/
+    --expected_output expected_cmdline_output_files/livesim_test_all_txt.txt
+  expected: 0
+
```

Comment:
When running this test locally, I get unexpected output:
```
global: cluster has 6 unique endpoints, 2 namespaces None
./livesim_tests/livesim_topology.yaml:133:8: Warning: A namespaceSelector selects no pods. Better use "namespaceSelector: Null" <_io.TextIOWrapper name='<stderr>' mode='w' encoding='utf-8'>
./livesim_tests/livesim_topology.yaml:131:4: Warning: Rule selects no pods <_io.TextIOWrapper name='<stderr>' mode='w' encoding='utf-8'>
Warning: No ingress-nginx pods found, the Ingress policy will have no effect <_io.TextIOWrapper name='<stderr>' mode='w' encoding='utf-8'>
Warning: selector ordereddict([('istio', 'ingressgateway')]) does not reference any pods in Gateway istio-gateway. Ignoring the gateway <_io.TextIOWrapper name='<stderr>' mode='w' encoding='utf-8'>
Warning: no valid Gateways found. Ignoring istio ingress traffic <_io.TextIOWrapper name='<stderr>' mode='w' encoding='utf-8'>
error: unexpected outputFormat in output configuration value [should be txt/yaml/csv],  value is: ./txt/
./txt/ output format is not supported for this query
```

Reply:
fixed